db/schema.rb

```diff
@@ -13,7 +13,6 @@
 ActiveRecord::Schema[7.0].define(version: 2023_03_07_203559) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "pg_stat_statements"
-  enable_extension "pgcrypto"
```

Comment:
How is deleting pgcrypto related to this PR? Was it added for the combined hybrid handoff?

Reply:
I'm not sure how this happened. Could it have been the result of merging in `main` at some point?

Reply:
I think it's my local database, I'll revert this